mod/admin.php

```diff
@@ -1590,14 +1781,14 @@ function admin_page_logs(&$a){
 			}
 			fclose($fp);
 		}
-	}
+	}*/
 
```

Comment:
It moved in its own function as config and display of logs are separated now.

I should rather remove the commented code (actually I thought I had).